prom/tr/generic.go

```diff
@@ -36,9 +36,13 @@ func (tr Generic) Translate(values []blip.MetricValue, ch chan<- prometheus.Metr
 		var promType prometheus.ValueType
 		var help string
 		switch values[i].Type {
-		case blip.COUNTER:
+		case blip.CUMULATIVE_COUNTER:
 			promType = prometheus.CounterValue
 			help = "Generic counter metric."
+		case blip.DELTA_COUNTER:
```

Comment:
We could do the reverse we are doing in Datadog sink (maintaining a cumulative counter). I am leaving it for a future change?

Reply:
Afaik, Prom send cumulative counters. E.g. last time I looked at what msyqld_exporter did, it was raw numbers from MySQL (i.e. cumulative counters).

Reply:
https://prometheus.io/docs/concepts/metric_types/

Reply:
Yes, prom send Cumulative counters. As this was switching on Metric Types, I need to include `DELTA_COUNTER` as a case for completeness.

I am ignoring delta values for platforms which don't support them atm. But we could do reverse of what we are doing in Datadog sink to accumulate deltas.